slack_messaging.gs

```diff
@@ -64,36 +63,52 @@ var postToSlack = function(payload, url, as_user=false){
         .getContentText();
 }
 
+class Display {
```

Comment:
I'm not sure how I feel about this being a class. The idea of separating out the repeated functionality is a good one, but by subclassing all the displays in this way, it suggests that they are exchangeable. For example you can't use an `AdminDisplay` in place of a `TrackingSheetDisplay`, because `AdminDisplay` expects a valid `payload`, not just a `msg`.

What do you think?

There is also

Reply:
You're right. I tried to rework things in my [latest commit](https://github.com/romseymutualaid/helpRequests/pull/24/commits/4f16044407ebe56cd76f2e262427ea19e1b83a96).